bench.ps1

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env pwsh
+# Add `--filter *.METHOD*` or similar to run subset of benchmarks
+dotnet run -c Release --project NCsvPerf\NCsvPerf.csproj -- -m --minWarmupCount 3 --maxWarmupCount 5 --minIterationCount 5 --maxIterationCount 11
```

Comment:
This will take a long time to run I think on my PC, but I guess that's fine. We'll see how it goes next time I update the blog. The original `SimpleJob` parameters were a nice balance between stability and fast runtime. I don't feel strongly.

Reply:
I actually just tweaked the params based on some more running. I underestimated how slow some of the entries are... I'll submit another PR 😅